gnetlist/src/s_traverse.c

```diff
@@ -440,6 +440,6 @@ s_init_traverse ()
 {
   /* Define the (gsymcheck core check) module */
   scm_c_define_module ("gnetlist core traverse",
-                       init_module_gnetlist_core_traverse,
```

Comment:
Maybe it is "more correct" to change the definition of `init_module_gnetlist_core_traverse()` to:

```C
static void
init_module_gnetlist_core_traverse (void *unused)
{
    /* ... */
}
```